osu.Game/Overlays/Dashboard/CurrentlyPlayingDisplay.cs

```diff
@@ -35,35 +37,42 @@ internal class CurrentlyPlayingDisplay : CompositeDrawable
 
         private SearchContainer<PlayingUserPanel> userFlow;
 
-        private FocusedTextBox searchBar;
-        private Container<FocusedTextBox> searchBarContainer;
+        private Box searchBarBackground;
+        private BasicSearchTextBox searchBar;
+        private Container<BasicSearchTextBox> searchBarContainer;
 
         [Resolved]
         private SpectatorClient spectatorClient { get; set; }
 
         [BackgroundDependencyLoader]
-        private void load()
+        private void load(OverlayColourProvider colourProvider)
         {
             RelativeSizeAxes = Axes.X;
             AutoSizeAxes = Axes.Y;
 
-            searchBarContainer = new Container<FocusedTextBox>
+            searchBarBackground = new Box
+            {
+                RelativeSizeAxes = Axes.X,
+                Height = 10*2 + search_bar_height,
```

Comment:
```suggestion
                Height = 10 * 2 + search_bar_height,
```

Missing spaces. But also, `10 * 2` means the same as `20` unless you split the `10` out into a named constant. So choose one of those two options.

Reply:
also done, decided to use a padding constant!